fern/docs.yml

```diff
@@ -240,6 +240,9 @@ navigation:
         snippets:
           python: "octoai"
           typescript: "@octoai/sdk"
+      - api: Media Utilities API
+        api-name: image-gen-util
```

Comment:
I left it called image-gen-util to keep it consistent with original filename. Can rename to `media-util` if we prefer